sdk/monitor/opentelemetry-exporter/src/Declarations/Contracts/Generated/RequestData.ts

```diff
@@ -0,0 +1,55 @@
+// THIS FILE WAS AUTOGENERATED
+import Domain = require('./Domain');
+'use strict';
+
+/**
+ * An instance of Request represents completion of an external request to the application to do work and contains a summary of that request execution and the results.
+ */
+class RequestData extends Domain {
+  /**
+   * Schema version
+   */
+  public ver: number;
+
+  /**
+   * Identifier of a request call instance. Used for correlation between request and other telemetry items.
+   */
+  public id: string | undefined;
+
+  /**
+   * Source of the request. Examples are the instrumentation key of the caller or the ip address of the caller.
+   */
+  public source: string | undefined; // @todo
```

Comment:
what's the `// @todo` for?

Reply:
This field isn't currently set by this exporter or the old SDK. I need to figure what fields map to source. Updated the todo comment here.